### DISCUSSION TITLE
MF6 OBS Package

### DISCUSSION THREAD

**Discussion by tjfarnum** [gmdsi/GMDSI_notebooks#99] [General]
@mnfienen 

My understanding is that the built in OBS package for Modflow6 only returns the head value at the center of the cell.  The older HOBS package for 2005/NWT would do some linear interpolation from the head at the center of the cell to the exact location of a head target.  John Doherty’s utility MF6MOD2OBS does a similar linear interpolation to get a more exact value at the location of the head target and more importantly works with Modflow6.
 
For small(ish) cell sizes the difference from the center of the cell to the edge of the cell might be small, but for larger cells and a steep head gradient, this could potentially be meters off.  I guess my questions would be:
 
1) Is there a reason Modflow6 OBS went away from linear interpolation to the exact location?
2) Instead of using the built in OBS package with Pyemu, would it make more sense to instead make the “model run” file incorporate ALSO running MF6MOD2OBS and getting a more exact head value to compare the sum of squares against?

**Comment by mnfienen** [comment]
great question, @tjfarnum. It's my understanding that not providing interpolation for head obs is intentional from the MF6 development team. There is sometimes debate about whether linear interpolation should be performed since it can result in changes in values that aren't really based on the physics of the model.

Nonetheless, for sure it's sometimes useful, so tools like MF6MOD2OBS are a good bet to use.  Putting such post-processing steps  into the model run file is a great solution and often is part of the process. Another example is lining up model observations with specific times, or things like transmissivity weighting across multiple model layers.

There will be an upcoming integration of some of John Doherty's  utilities into pyemu in the near future - stay tuned!
